apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -88,13 +88,20 @@ export type BaseNewsFormProps = {
 }
 
 const NO_RESIZE_STYLE: React.CSSProperties = { resize: 'none' }
-const FORM_FILED_COL_PROPS = { style: { width: '100%', padding: 0 } }
+const FLEX_START_STYLE: React.CSSProperties = { alignItems: 'flex-start' }
+const BIG_MARGIN_BOTTOM_STYLE: React.CSSProperties = { marginBottom: '60px' }
+const MARGIN_BOTTOM_32_STYLE: React.CSSProperties = { marginBottom: '32px' }
+const MARGIN_BOTTOM_38_STYLE: React.CSSProperties = { marginBottom: '38px' }
+const MARGIN_BOTTOM_10_STYLE: React.CSSProperties = { marginBottom: '10px' }
+const MARGIN_BOTTOM_24_STYLE: React.CSSProperties = { marginBottom: '24px' }
+const MARGIN_TOP_8_STYLE: React.CSSProperties = { marginTop: '8px' }
+const MARGIN_TOP_44_STYLE: React.CSSProperties = { marginTop: '44px' }
```

Comment:
Can you do this with Row gutter or our Space component from ui kit?

Reply:
no because of the default styles of antd components, for example form.item labels I need to set different margins in different parts of the application to make the layout match the design layout. 
Previously it was done through Row gutter but the design team was not very satisfied so I had to manually rearrange the values to make the layout more pixel perfect)

Reply:
But in other forms, we could handle the indentation without a bunch of custom margins.
I think it's better to wrap form label with 0 margin and use [default spacing](https://www.figma.com/file/Nbkug4IRM33sEyDuN0yO1C/UI-Kit-(web)?node-id=1240%3A5095&mode=dev)

Reply:
Would it be a comfortable solution for both of us if I create new task  on code refactoring and add a TODO comment to the code with the number of this task?

At this moment I have tasks with higher priority and users are seeing a different interface than the design intended. I propose to upload these fixes unchanged with not hight quality code so that users can enjoy the beauty rn and rewrite the code later, when I have free slots for tasks